packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcher.tsx

```diff
@@ -14,11 +14,13 @@ const _OrganizationSwitcher = withFloatingTree(() => {
 
   return (
     <Flow.Root flow='organizationSwitcher'>
-      <OrganizationSwitcherTrigger
-        ref={reference}
-        onClick={toggle}
-        isOpen={isOpen}
-      />
+      <Flex>
```

Comment:
Why is this change needed?

Reply:
@nikosdouvlis https://github.com/clerkinc/javascript/pull/550#discussion_r1032393958